### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by guillenotfound**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ async function subsystem (fastify, opts) {
 
 To use this module with TypeScript, make sure to install `@types/express`.
 
+In addition, you will need to add `"types": ["fastify-express"]` to your tsconfig.json file when using `require` to import the plugin, otherwise it will work out of the box.
```
Comment (Fdawgs) [inline README.md:154]: Wouldn't it be "otherwise it will not work out of the box"?

Comment (guillenotfound) [reply README.md:154]: if (require) "add types"
else "will work"

I think is well written, do you prefer to say it with different words?

Comment (Fdawgs) [inline README.md:154]:
```suggestion
You will need to add `"types": ["fastify-express"]` to your tsconfig.json file when using `require` to import the plugin.
```

**Review by Fdawgs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Fdawgs) [reply README.md:154]: See other response, just needed cutting down.

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**